.github/workflows/checks.yml

```
@@ -46,7 +46,9 @@ jobs:
           path: ~/.tools
           key: ${{ runner.os }}-${{ env.cache-name }}-${{ hashFiles('./internal/tools/**') }}
       - name: Run linters
-        run: make license-header-check
+        run: make license-header-check go-mod-tidy
```

How about simplifying to the following?

```suggestion
        run: make precommit
```

---

This is going to be different than `precommit`, see https://github.com/open-telemetry/opentelemetry-go/blob/86f325839ca329997672fbae8b054e7b6181b4d2/.github/workflows/ci.yml#L47

---

I see. How about encapsulating it to a `ci` Make target?

---

Make sense. Resolving.